test/parallel/test-stream-map.js

```diff
@@ -86,6 +86,10 @@ const { setTimeout } = require('timers/promises');
 
 {
   // Error cases
+  assert.rejects(async () => {
+    // Validation errors thrown synchronously
+    Readable.from([1]).map(1);
+  }, /ERR_INVALID_ARG_TYPE/).then(common.mustCall());
   assert.rejects(async () => {
     // eslint-disable-next-line no-unused-vars
     for await (const unused of Readable.from([1]).map(1));
```

Comment:
All these would presumably have to be fixed right? (If they are not needed because the check is duplicated - they can be removed).

Additionally - instead of `assert.rejects` here you can use `assert.throws(() => Readable.from([1]).map(1), /ERR_INVALID_ARG_TYPE/)` directly which would save the `common.mustCall()` as well.

Reply:
was wondering about that. np, will do.